### ISSUE TITLE
Run a thorough IBM DAP analysis

### ISSUE THREAD

**Issue by theiliad** [IBM/carbon-components-angular#52]
There are some violations in the components currently.

**Keep in mind:**
DAP throws many false violations in there, and some violations are caused by the storybook templates.

https://www.ibm.com/able/dynamic-assessment-plug-in.html
